src/BottomNavigation/BottomNavigation.react.js

```diff
@@ -44,20 +51,61 @@ function getStyles(props, context) {
 */
 
 class BottomNavigation extends PureComponent {
+    constructor() {
+        super();
+        this.state = {
+            moveAnimated: new Animated.Value(0),
+        };
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.hidden !== this.props.hidden) {
+            if (nextProps.hidden === true) {
+                this.hide();
+            } else {
+                this.show();
+            }
+        }
+    }
+
+    show = () => {
+        Animated.timing(this.state.moveAnimated, {
+            toValue: 0,
+            duration: 225,
+            easing: Easing.bezier(0.0, 0.0, 0.2, 1),
+            useNativeDriver: Platform.OS === 'ios',
+        }).start();
+    }
+
+    hide = () => {
+        Animated.timing(this.state.moveAnimated, {
+            toValue: this.context.uiTheme.bottomNavigation.container.height,
```

Comment:
This is not correct. Because it always returns `undefined`. Another problem is when you override `...container.height` it will return bad number.

Ends with this error:
<img width="407" alt="screen shot 2017-01-03 at 7 40 18 pm" src="https://cloud.githubusercontent.com/assets/3531955/21618697/8238c1aa-d1ec-11e6-95e9-d8eddc97f073.png">